react/features/calendar-sync/functions.any.js

```diff
@@ -89,40 +89,35 @@ export function _updateCalendarEntries(events: Array<Object>) {
 function _parseCalendarEntry(event, knownDomains) {
     if (event) {
         const url = _getURLFromEvent(event, knownDomains);
-
-        // we only filter events without url on mobile, this is temporary
-        // till we implement event edit on mobile
-        if (url || navigator.product !== 'ReactNative') {
-            const startDate = Date.parse(event.startDate);
-            const endDate = Date.parse(event.endDate);
-
-            // we want to hide all events that
-            // - has no start or end date
-            // - for web, if there is no url and we cannot edit the event (has
-            // no calendarId)
-            if (isNaN(startDate)
-                || isNaN(endDate)
-                || (navigator.product !== 'ReactNative'
-                        && !url
-                        && !event.calendarId)) {
-                logger.debug(
-                    'Skipping invalid calendar event',
-                    event.title,
-                    event.startDate,
-                    event.endDate,
-                    url,
-                    event.calendarId
-                );
-            } else {
-                return {
-                    calendarId: event.calendarId,
-                    endDate,
-                    id: event.id,
-                    startDate,
-                    title: event.title,
-                    url
-                };
-            }
+        const startDate = Date.parse(event.startDate);
+        const endDate = Date.parse(event.endDate);
+
+        // we want to hide all events that
+        // - has no start or end date
+        // - for web, if there is no url and we cannot edit the event (has
+        // no calendarId)
+        if (isNaN(startDate)
+            || isNaN(endDate)
+            || (navigator.product !== 'ReactNative'
+                    && !url
+                    && !event.calendarId)) {
```

Comment:
Don't we need to get rid and from this ReactNative check? 
And then events without url and calendarId can be shown, those without url can be edited.

Reply:
I'm not sure about that, that is web code (```navigator.product !== 'ReactNative'```). But it looks good to me: if it's web, and there is no URL and there is no calendarId, then we don't show that because we can't join that meeting and can't even update it (as there is no calendarId). Is it what you mean?

Reply:
On mobile we don't need to check the calendarID, because we don't have such info